cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonConstants.java

```diff
@@ -68,6 +68,7 @@ private CommonConstants() {
     public static final String LOG4J_COMP = "agent";
     
     // encoding
+    public static final String INTERNAL_CONCATENATOR = "=";
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1057/commits/e70d1dfa29ba6dc4c20f2a649f115a985d585a36